modules/users/server/controllers/users/users.profile.server.controller.js

```diff
@@ -111,7 +111,15 @@ exports.changeProfilePicture = function (req, res) {
       if (existingImageUrl !== User.schema.path('profileImageURL').defaultValue) {
         fs.unlink(existingImageUrl, function (unlinkError) {
           if (unlinkError) {
-            console.log(unlinkError);
+
+            // If file didn't exist, no need to reject promise
+            if (unlinkError.code === 'ENOENT') {
+              console.log('Removing profile image failed because file did not exist.');
+              return resolve();
+            }
+
+            console.error(unlinkError);
```

Comment:
> I'm wondering if you maybe want to use the logger instead of console.log/error

Right on, I forgot meanjs has one.

>what do you mean it wont be logged?

Sorry, I meant won't be logged in _error log_ since function gets returned before that line.